packages/firestore/src/api/database.ts

```diff
@@ -653,31 +650,18 @@ export class Firestore implements firestore.FirebaseFirestore, FirebaseService {
     return new WriteBatch(this);
   }
 
-  static get logLevel(): firestore.LogLevel {
```

Comment:
Isn't this a breaking change? I guess `logLevel` was never in the d.ts file so typescript users couldn't have counted on this. What about plain JavaScript users? Was this previously mangled?

Reply:
It exists in the mangled build:

Object.defineProperty(t, "logLevel", {
        get: function() {
            switch (Hn()) {
              case 0 /* DEBUG */ :
                return "debug";

              case 1 /* ERROR */ :
                return "error";

              case 2 /* SILENT */ :
                return "silent";

              default:
                return ti("Unknown log level: " + Hn());
            }
        },

It was never part of our types and not in the public API, but it also wasn't prefixed with an underscore. Let me sleep on this.

Reply:
I added it back, but also added an entry to the "breaking changes" doc to either remove it make it part of the public API.